docs/getting_started.rst

```diff
@@ -139,9 +139,31 @@ Alternatively, to create just the output file or directory of one rule, run:
 
      snakemake -j<NUMBER_OF_CPU_CORES> <output file or folder>
 
+If needed, all calculated scenarios in the directory :file:`results` can be deleted with the following rule:
+
+::
+
+    snakemake -j1 clean
 
 Contributing to oemof-B3
 ========================
 
+In case you want to develop your own scenarios and calculations based on oemof-B3,
+by perhaps adding components, there are two additional snakemake rules for the modified energy system.
+Executing the rule
+
+::
+
+    snakemake -j1 create_empty_scalars
+
+will create empty scalars. The rule
+
+::
+
+    snakemake -j1 create_empty_ts
+
+will create empty time series data.
+The empty scalars and time series data can be used to perform a quick test run and verify your energy system model.
+
```

Comment:
This is not quite true. `var_value` needs to be set in the empty scalars to have a test run. Time series are ready to go, since zeros are added by default. In the private repo `digipipe` I work on a script that sets default scalars. Maybe we can add it to oemof-B3 after it's merged there.

Could you please adapt the text for now with the info?

Reply:
Done with commit 08c6ef4ffa121991adeb20ad2767c6b9053e89d6